### DISCUSSION TITLE
account suspensed

### DISCUSSION THREAD

**Discussion by arvi0011** [community/community#46580] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

about 10 days ago i logged in to github to make a donation on gitcoin, i logged in using the laptop i just bought from my friend, that time the account was still active and smooth, 5 days later i logged in to github in the same browser and it's not working and it says "Your browser did something unexpected. Please try again. If the error persists, try disabling all browser extensions."
after i reset my browser my account was suspended,
my question:
did I make mistake?
is there any solution to open a suspended account?
Thank You

**Comment by jge162** [comment, marked as answer]
You need to reach out to [support](https://support.github.com/request/account?tags=hubberfy_account)

**Reply by arvi0011** [reply]
ok thank

**Reply by jge162** [reply]
Choose sign in issues

**Reply by queenofcorgis** [reply]
Support are the only folks that can help you in this matter so I'll be locking this post as we're unable to assist further. Best of luck.
